client/app/pages/dashboards/dashboard.js

```diff
@@ -249,6 +251,11 @@ function DashboardCtrl(
     });
   };
 
+  this.downloadPdf = () => {
```

Comment:
Have to check if this even runs when a click on the download button starts the download anyway.